packages/java-edition/src/mcfunction/node/argument.ts

```diff
@@ -119,7 +120,7 @@ export namespace EntitySelectorArgumentsNode {
 		)
 	}
 }
-export const EntitySelectorVariables = ['a', 'e', 'p', 'r', 's'] as const
+export const EntitySelectorVariables = ['a', 'e', 'p', 'r', 's', 'n'] as const
```

Comment:
(pretty minor) I think we could remove the `export ` from `EntitySelectorVariables` and `EntitySelectorAtVariables` and add a comment to note that `The filterAvailable() function should be used to get a list of available selector variables` to avoid any future mistakes?